package/MDAnalysis/coordinates/base.py

```diff
@@ -1741,7 +1749,66 @@ def get_aux_descriptions(self, auxnames=None):
             auxnames = self.aux_list
         descriptions = [self._auxs[aux].get_description() for aux in auxnames]
         return descriptions
-
+    
+    @property
+    def transformations(self):
+        """ Returns the list of transformations"""
+        return self._transformations
+    
+    @transformations.setter
+    def transformations(self, transform_list):
+        if not self._transformations:
+            self._transformations = transform_list
+        else:
+            raise ValueError("Transformations are already set")
+        
+    def add_transformations(self, transform_list):
```

Comment:
Hadn't @orbeckst said something about not using types in variable names? `transform_list` could just well be `transformations`.

Also, I'd vote for an `*args` solution here; I think it's more pythonic. It's somewhat unintuitive for the user to have to pass single transforms in brackets:
```python
workflow = [some_transform, another_transform, this_transform]

# Now
u.trajectory.add_transformations([just_one_transform])
# or
u.trajectory.add_transformations(workflow)

# With *args
u.trajectory.add_transformations(just_one_transform)
# or
u.trajectory.add_transformations(*workflow)
```

Reply:
+1 for the `*args`. Having the variable named `transformed_list` is not a big deal here, though: the variable actually points to a list. It was more of an issue when it was pointing at a dict.